code/modules/mob/living/say.dm

```diff
@@ -48,7 +48,7 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 
 	// Faction
 	"å" = "Syndicate",
-	"í" = "CentCom",
+	"í" = "Horizons",

```

Comment:
This should be `Head Office`, or whatever you set the define to before. I think.